dist/index.html

```diff
@@ -2,19 +2,183 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>My JavaScript App</title>
+  <title>BackTREK </title>
+  <!--[if lt IE 9]>
+  <script src = "http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+  <![endif]-->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" type="image/ico" href="/favicon.ico">
 </head>
 <body>
-  <header>
+  <div class="container-fluid body-wrapper">
+    <header class="row header">
+      <h1 class="col">BackTREK</h1>
+    </header>
 
-  </header>
-  <main>
+    <main class="row clearfix main">
+      <!-- <section id="status" class="col-12">
+        <div class="button-wrapper row">
+          <i class="fa fa-close fa-lg"></i>
+        </div>
+        <ul class="msg-list row">
+          <li>error or something</li>
+        </ul>
+      </section> -->
 
-  </main>
-  <footer>
+      <section class="col-12 main-content">
+        <div class="row content-wrapper">
+          <section class="col-sm-12 col-md-6 all-trips">
+            <div class="row search-wrapper">
+              <form class="col-sm-12 form-inline" action="">
```

Comment:
You don't seem to have any JavaScript catching this form's submit action, so it causes a page reload.